spec/EventSubscriber/HiddenLocationExceptionSubscriberSpec.php

```diff
@@ -4,17 +4,18 @@
  * @copyright Copyright (C) Ibexa AS. All rights reserved.
  * @license For full copyright and license information view LICENSE file distributed with this source code.
  */
+
```

Comment:
I guess it's not covered by existing Rector rules, but would be good to add `declare(strict_types=1);` whenever possible as we touch such amount of codebase...

Reply:
`strict_types` do have an effect on the execution, so we'd want this done in more controlled fashion than just adding it everywhere.

Reply:
>whenever possible

I am not talking about introducing a specific rector, but decreasing amount of code to touch on our way to Sf6 whenever a low-hanging fruit kind of situation presents itself.